src/main/java/com/fjb/sunrise/services/impl/UserServiceImpl.java

```diff
@@ -16,25 +17,33 @@
 public class UserServiceImpl implements UserService {
     @Value("${default.admin-create-key}")
     private String key;
+    private final UserMapper mapper;
     private final UserRepository repository;
```

Comment:
In the event that we inject various repositories, it might be prudent to use 'userRepository' instead of a generic 'repository' to ensure code stability in the future.